test/DebugInfo/debug_prefix_map_sdk.swift

```diff
@@ -0,0 +1,8 @@
+// RUN: %empty-directory(%t)
+// RUN: %target-swift-frontend -sdk %t -g %s -emit-ir | %FileCheck -check-prefix=WITHOUT-PREFIX-MAP %s
+// RUN: %target-swift-frontend -sdk %t -g %s -emit-ir -debug-prefix-map %t=foo | %FileCheck -check-prefix=WITH-PREFIX-MAP %s
+
+func foo() {}
+
+// WITHOUT-PREFIX-MAP: sysroot: "BUILD_DIR
```

Comment:
This check looks to me like it can't possibly work. We don't know what %t expands to, do we?

How about
` -sdk %t/SOME_RECOGNIZEABLE_STRING`

`// WITHOUT-PREFIX-MAP: sysroot: "{{.*}}SOME_RECOGNIZEABLE_STRING`

Reply:
This works locally because it expands to some absolute path, and then lit converts the path to the test to `BUILD_DIR`, are you saying I shouldn't rely on that? The key here is having it verify the absolute string though so I don't think your example would verify what we want? Unless we make a nested directory and prefix map that whole thing, which would probably work. Previously I was doing `sysroot: "/` but that won't work because of the `BUILD_DIR` replacement.

Reply:
Ah I understand what you're suggesting, fixing now

Reply:
Updated!